src/Uno.UI/UI/Xaml/FrameworkElement.wasm.cs

```diff
@@ -204,26 +204,29 @@ internal void RaiseSizeChanged(SizeChangedEventArgs args)
 
 		protected void SetCornerRadius(CornerRadius cornerRadius)
 		{
-			var borderRadius = cornerRadius == CornerRadius.None
-				? ""
-				: $"{cornerRadius.TopLeft}px {cornerRadius.TopRight}px {cornerRadius.BottomRight}px {cornerRadius.BottomLeft}px";
+			if (cornerRadius == CornerRadius.None)
+			{
+				ResetStyle("border-radius", "overflow");
+			}
+			else
+			{
+				var borderRadiusCssString =
+					$"{cornerRadius.TopLeft.ToStringInvariant()}px {cornerRadius.TopRight.ToStringInvariant()}px {cornerRadius.BottomRight.ToStringInvariant()}px {cornerRadius.BottomLeft.ToStringInvariant()}px";
+				SetStyle(
+					("border-radius", borderRadiusCssString),
+					("overflow", "hidden"));
```

Comment:
Would be useful to have a comment on this line.

Reply:
Also: what was the original motivation for removing `overflow: hidden`? Will there be weird interactions with `CornerRadius` in some cases?

Reply:
It wasn't useful for clipping since we're already doing it elsewhere using the clipping mask, which will also consider the `.Clip` property of the element. In that case, the overflow: hidden is useless.

The only place were it's useful is when we're having a non-rectangular clipping masks. And the only place where it could happend on UWP is by using the corner radius. [because the `.Clip` only accepts rectangles].